Lib/fontTools/ttLib/tables/_g_v_a_r.py

```diff
@@ -23,19 +23,24 @@
 # FreeType2 source code for parsing 'gvar':
 # http://git.savannah.gnu.org/cgit/freetype/freetype2.git/tree/src/truetype/ttgxvar.c
 
-GVAR_HEADER_FORMAT = """
+GVAR_HEADER_FORMAT_HEAD = """
 	> # big endian
 	version:			H
 	reserved:			H
 	axisCount:			H
 	sharedTupleCount:		H
 	offsetToSharedTuples:		I
-	glyphCount:			H
+"""
+# In between the HEAD and TAIL lies the glyphCount, which is
+# of different size: 2 bytes for gvar, and 3 bytes for GVAR.
+GVAR_HEADER_FORMAT_TAIL = """
+	> # big endian
 	flags:				H
 	offsetToGlyphVariationData:	I
 """
```

Comment:
I feel that duplicating `GVAR_HEADER_FORMAT` for each table would make the code simpler. May be have a `GVAR_HEADER_FORMAT_TEMPLATE` or some such that has `glyphCount: %s`, then the table class would have something like `GVAR_HEADER_FORMAT = GVAR_HEADER_FORMAT_TEMPLATE % gid_size`.

Reply:
I think the problem is that there's no way to read 24bit using `sstruct`. No?

Reply:
I didn’t know that. OK then.